fr/common.json

```
@@ -0,0 +1,452 @@
+    }
+  },
+  "doubleSidedVariationChart": {
+    "inactive": "NC",
+    "inactiveMMR": "NC",
+    "event": {
+      "version": "PATCH {{version}}"
+    },
+    "tooltip": {
+      "characterWeapon": "{{characterWeapon}} / {{metrics}}: {{value}}",
+      "mmrFlow": "Dernière MMR: {{last}} (max.: {{max}}, min.: {{min}})"
+    }
+  },
+  "widget": {
+    "playerNotFound": "Joueur Introuvable",
+    "recent3Matches": "3 dernières parties",
+    "nextAutoUpdate": "Mise à jour dans {{time, seconds}}",
+    "updating": "Mise à jour..."
+  },
+  "masteryImprovement": {
+    "average": "Moyenne",
+    "perMinute": "{{number}}/min."
+  },
+  "itemTooltip": {
+    "gather": "Récolte",
+    "monster": "Chasse",
+    "area": "Zone",
+    "supply": "Colis Aérien",
+    "specificCharacterOnly": "Fabricable par {{name}}",
+    "maxStackCount": "Quantité max",
+    "initialCount": "Quantité initiale",
+    "buildsInto": "Transformable en",
+    "bulletCapacity": "Munitions max",
+    "bulletCapacityValue": "{{count}}",
+    "chargeCapacity": "Charges max",
+    "chargeCapacityValue": "{{count}} (recharge : {{chargeTime, seconds}})"
+  },
+  "textTooltip": {
+    "tier": "Le rang représente la puissance et la popularité du personnage, compte tenu du taux de victoire, le pourcentage de classement moyen, les éliminations, les modifications du MMR et le taux de mortalité. Il peut différer même si le Personnage est identique selon les échantillons.",
+    "perishRate": "Le taux de mortalité représente la fréquence de mort en début de partie, compte tenu de la durée de la partie, le niveau d'arme, les éliminations.",
+    "halfRate": "Le pourcentage de classement moyen représente le classement normalisé du joueur dans un match. Si le pourcentage de classement moyen d'un joueur est de 50 %, cela signifie que le classement moyen est respectivement 9,5e, 5e ou 3,5e en solo, en duo ou en escouade.",
+    "artisan": "Le score d'artisanat représente à quel point le personnage a été utilisé par le joueur dans les parties à haut niveau. Contrairement à la MMR, celui-ci ne diminue jamais.",
+    "dpmToCharacter": "Les DPM (dégâts par minute) représentent les dégâts infligés aux autres joueurs par minute."
+  },
+  "tierBadge": {
+    "op": "OP",
+    "up": "RIP",
+    "a": "Expert",
+    "lowUniquePlayers": "< {{lowCutoff}}",
+    "lowUniquePlayersTooltip": "Ces données ont été calculées avec des parties de seulement {{count}} joueurs,\nce qui signifie qu'elles peuvent facilement être influencées par le style de jeu de chaque joueur."
+  },
+  "characterPicker": {
+    "searchPlaceholder": "Chercher...",
+    "statisticsNotAvailable": "Données insuffisantes."
+  },
+  "internalBanner": {
+    "route": {
+      "first": "\"Si seulement je pouvais trouver le trajet optimal pour les objets que je veux !\"",
+      "second": "\"Vais-je manquer de place dans mon inventaire ?\"",
+      "thirdBefore": "La réponse dans le",
```

As you can see this at the home page, it should have a trailing space for the next field.
```suggestion
      "thirdBefore": "La réponse dans le ",
```